packages/template-retail-react-app/app/commerce-api/hooks/useBasket.js

```diff
@@ -392,6 +392,27 @@ export default function useBasket(opts = {}) {
                 return api.shopperBaskets.getShippingMethodsForShipment({
                     parameters: {basketId: basket.basketId, shipmentId: 'me'}
                 })
+            },
+
+            /**
+             * Merge data from the previous shopper's basket into the current shopper's active basket
+             * and delete the previous shopper's basket.
+             */
+            async mergeBasket() {
+                const response = api.shopperBaskets.mergeBasket({
+                    headers: {
+                        'Content-Type': 'application/json'
+                    },
```

Comment:
```suggestion
```
Assuming this calls the isomorphic sdk, I don't think it's necessary to set this header here.

Reply:
I too thought the sdk would handle this but the API returns `Unexpected payload type` error if the header is removed. I can move it to `ShopperBaskets` class where the sdk function is called though.